src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3VettingHelper.java

```diff
@@ -59,43 +66,69 @@ public static boolean isThereAnEt3Response(CaseData caseData) {
             return false;
         }
 
+        String respondentName = caseData.getEt3ChooseRespondent().getSelectedLabel();
         for (RespondentSumTypeItem respondentSumTypeItem : respondentCollection) {
             RespondentSumType respondent = respondentSumTypeItem.getValue();
-            if (!isNullOrEmpty(respondent.getResponseReceived()) && respondent.getResponseReceived().equals(YES)) {
+            if (respondentName.equals(respondent.getRespondentName()) && YES.equals(respondent.getResponseReceived())) {
                 return true;
             }
         }
```

Comment:
Very Minor: This could be achieved using lambda.  Something like:
return respondentCollection
.stream()
.map(r->respondantName.equals(r.getValue().getRespondentName() && YES.equals(r.getValue().getResponseReceived())
.findFirst() != null;

It's slightly more elegant and more readable this way, but happy with the way you've done it too ;)